RELEASES.md

```diff
@@ -1,3 +1,105 @@
+Version 1.23.0 (2018-01-04)
+==========================
+
+Language
+--------
+- [Fixed displaying duplicate errors on some trait bounds.][45772]
+
+Compiler
+--------
+- [Enabled `TrapUnreachable` in LLVM which should prevent some cases of
+  undefined behaviour.][45920]
```

Comment:
It would be more accurate to say that it mitigates the impact of the UB. This change doesn't make any programs defined that were previously UB, it just makes it more likely they trap at runtime instead of executing arbitrary code.